osu.Game/Screens/Edit/Editor.cs

```diff
@@ -168,6 +168,8 @@ private void load(OsuColour colours, GameHost host)
 
             menuBar.Mode.ValueChanged += onModeChanged;
 
+            host.Exiting += onExitingGame;
```

Comment:
```suggestion
            host.Exiting += _ => isExitingGame = true;
```

Reply:
It was like this at first, but I changed that, so there is no way to change something in the line where we're subscribing, but not in the line where we're unsubscribing, which would obviously mess up stuff. Do you think I should change it anyway, for the sake of the code readability?

Reply:
do we need to unsubscribe at all?

Reply:
It subscribes in `load()`. `load()` runs every time we open the editor. From my understanding we would be adding to the event over and over again when entering and leaving the editor. Or perhaps I am mistaken?

Reply:
no, `load()` runs once, it's because a new `Editor` instance is created every time you enter the editor, so you're not subscribing everytime

Reply:
this is irrelevant because this code cannot exist (see my review).

also @iiSaLMaN you are wrong. not unsubscribing would result in a permanent memory leak.

Reply:
oh sorry